src/elements/TableCell/TableCell.js

```diff
@@ -182,6 +182,7 @@ const TableCell = ({
     return (
       <td className={cellClassNames}>
         <Tooltip
+          id={data?.driftStatus ? data.driftStatus : ''}
```

Comment:
You can't just add any new property that you need to the common component.

Reply:
@pinis-gini-apps I mean you can't use `driftStatus` here. `TableCell` is a common component

Reply:
@mavdryk What is the alternative component to use for QA test-id in this case?

Reply:
@pinis-gini-apps The `id` must be specified in the same place where we generate the content